server/graphql/v2/object/Expense.ts

```diff
@@ -85,7 +85,7 @@ const Expense = new GraphQLObjectType({
           currencySource: {
             type: ExpenseCurrencySource,
             description: 'Source of the currency to express the amount. Defaults to the expense currency',
-            defaultValue: ExpenseCurrencySource.EXPENSE,
+            defaultValue: 'EXPENSE',
```

Comment:
The problem is, it's a GraphQL enum - not a TS enum. So the previous `defaultValue` was probably not working (which did not create a bug since there's no `else if (source === 'EXPENSE')`).

To use this object, we would need to do:
```suggestion
            defaultValue: ExpenseCurrencySource.getValues().find(({ value }) => value === 'EXPENSE'),
```

It seems more straightforward to directly pass the value.